### DISCUSSION TITLE
UMD format

### DISCUSSION THREAD

**Discussion by pallendes** [system-ui/theme-ui#1985] [Q&A] [answered]
Hi everyone,
I was wondering if there is a CDN or any place from where I can download theme-ui in UMD format so that I can load the library directly in the browser. In my team we are using micro-frontends with single-spa and we would like to load theme-ui only once and then share it across the micro-frontends

**Comment by lachlanjc** [comment, marked as answer]
Not that I'm aware of! The files we ship to npm are available via unpkg like https://unpkg.com/theme-ui@0.11.3/dist/theme-ui.cjs.prod.js (see all versions [here](https://unpkg.com/browse/theme-ui@0.11.3/dist/)) but I don't think we have a UMD build. PR welcome, we'll need to edit the Preconstruct settings [like this](https://preconstruct.tools/configuration)!
